config/template.config

```diff
@@ -16,6 +16,7 @@ params {
     // GRCh38 blacklist - /hot/resource/tool-specific-input/GRIDSS-2.13.2/GRCh38-BI-20160721/ENCFF356LFX.bed
     gridss_blacklist = "/hot/resource/tool-specific-input/GRIDSS-2.13.2/GRCh38-BI-20160721/ENCFF356LFX.bed"
     gridss_reference_fasta = "/hot/resource/tool-specific-input/GRIDSS-2.13.2/GRCh38-BI-20160721/Homo_sapiens_assembly38.fasta"
+    other_jvm_heap = 4.GB
```

Comment:
**suggestion**: Let's move this to default.config and document it in the README; typically users won't need to change it so it can live in default and user can add it if they need to change it